main.js

```diff
@@ -8,33 +8,50 @@ requestDogs.addEventListener("load", printDogCards);
 requestDogs.open("GET", "dogs.json");
 requestDogs.send();
 
-// Create a function that formats cards
-function buildDogCards(dataArray) {
+// Create a request for cats.json
+var requestCats = new XMLHttpRequest();
+requestCats.addEventListener("load", printCatCards);
+requestCats.open("GET", "cats.json");
+requestCats.send();
+
+// Create a function that formats dog cards
+function buildCards(dataArray) {
+	var cardsHTML = "";
 	for (var i = 0; i < dataArray.length; i++) {
 		var current = dataArray[i];
-		outputDiv.innerHTML += "<article class='card'>" +
-													 "<header class='card-header'>" +
-													 "<h3>" + current.name + "</h3>" + "</header>" +
-													 "<div class='card-body'>" +
-													 "<p>" + current.types[0].type + "<p>" +
-													 "<p>" + current.types[0].volumes[0].name + "</p>" +
-													 "<p>" + current.types[0].volumes[0].price + "<p>" +
-													 "</div></article>" +
-													 "<article class='card'>" +
-													 "<header class='card-header'>" +
-													 "<h3>" + current.name + "</h3>" + "</header>" +
-													 "<div class='card-body'>" +
-													 "<p>" + current.types[1].type + "<p>" +
-													 "<p>" + current.types[1].volumes[1].name + "</p>" +
-													 "<p>" + current.types[1].volumes[1].price + "<p>" +
-													 "</div></article>";
+		cardsHTML += "<article class='card'>" +
+								 "<header class='card-header'>" +
```

Comment:
Trying to line up your code like this doesn't really work with tabs.  As you can see on github. I'd stick with normal indentation with tabs. Also you are using `<TAB><TAB>...<TAB><SPACE>"<header class=...` here which is mixing tabs and spaces. If you insist on lining up, add a space to the prior line `cardsHTML +=<SPACE><SPACE>"<article class='card'>" +`